src/plugin/electron/electronconfirmcert.js

```diff
@@ -68,7 +68,6 @@ class Controller {
     if (this.scope) {
       const certs = /** @type {Array<!Electron.Certificate>} */ (this.scope['certs']);
       if (certs) {
-        certs.sort(sortCerts);
         this['cert'] = certs[0];
```

Comment:
It still might be nice to sort certificates. Default, then by expiry date from latest to earliest, so you get ones that might work before ones that probably won't.

Possibly this is already done somewhere else, if so, disregard.

Reply:
I was conflicted on this because it seems Chromium is already doing some internal sorting in the list provided to Electron. It seemed better to leave the list as-is.